helm/prometheus-rules/templates/platform/atlas/alerting-rules/prometheus-operator.rules.yml

```diff
@@ -113,10 +113,10 @@ spec:
       annotations:
         description: Prometheus operator in {{`{{`}} $labels.namespace {{`}}`}} namespace isn't ready to reconcile {{`{{`}} $labels.controller {{`}}`}} resources.
         opsrecipe: "prometheus-operator/"
-      expr: min by (cluster_id, installation, provider, pipeline, namespace, controller) (max_over_time(prometheus_operator_ready{app=~"prometheus-operator.*|kube-prometheus-.*"}[5m]) == 0)
+      expr: min by (cluster_id, installation, provider, pipeline, namespace, controller) (max_over_time(prometheus_operator_ready{job=~"prometheus-operator-app-operator|kube-prometheus-stack-operator"}[5m]) == 0)
```

Comment:
I guess this job's name changes with the renaming of `prometheus-operator` to `kube-stack-operator`?